cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -214,7 +215,58 @@ fn lower_insn_to_regs<'a>(ctx: Ctx<'a>, inst: IRInst) {
                 unimplemented!("unimplemented lowering for opcode {:?}", op);
             }
         }
-
+        Opcode::Fcopysign => {
+            let dst = output_to_reg(ctx, inst, 0);
+            let lhs = input_to_reg(ctx, inst, 0);
+            let rhs = input_to_reg(ctx, inst, 1);
+            let is64 = flt_ty_is_64(ty.unwrap());
+            if !is64 {
+                let w64 = 0x8000_0000;
+                let tregI64 = ctx.alloc_tmp(RegClass::I64, I32);
+                let tregV128 = ctx.alloc_tmp(RegClass::V128, F64);
+                let t2regV128 = ctx.alloc_tmp(RegClass::V128, F64);
+                let t3regV128 = ctx.alloc_tmp(RegClass::V128, F64);
+                ctx.emit(Inst::imm_r(true, w64, tregI64));
```

Comment:
Can you add a comment at the top of this sequence explaining how this is lowered, please? like:

```
mov 0x80000000, tregI64
movd tregI64, tregV128
```

etc